src/test/java/com/commercetools/sync/inventory/InventorySyncTest.java

```diff
@@ -0,0 +1,214 @@
+package com.commercetools.sync.inventory;
+
+import com.commercetools.sync.commons.helpers.CtpClient;
+import com.commercetools.sync.inventory.helpers.InventorySyncStatistics;
+import com.commercetools.sync.services.TypeService;
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.client.BlockingSphereClient;
+import io.sphere.sdk.client.SphereClientConfig;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+import io.sphere.sdk.models.Reference;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class InventorySyncTest {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(InventorySyncTest.class);
+
+    private final static String SKU_1 = "1000";
+    private final static String SKU_2 = "2000";
+    private final static String SKU_3 = "3000";
+
+    private final static String KEY_1 = "channel-key_1";
+    private final static String KEY_2 = "channel-key_2";
+    private final static String KEY_3 = "channel-key_3";
+
+    private final static String REF_1 = "111";
+    private final static String REF_2 = "222";
+    private final static String REF_3 = "333";
+
+    private static final String CTP_KEY = "testKey";
+    private static final String CTP_ID = "testId";
+    private static final String CTP_SECRET = "testSecret";
+
+    private final Long QUANTITY_1 = 10l;
+    private final Long QUANTITY_2 = 30l;
+
+    private final Integer RESTOCKABLE_1 = 10;
+    private final Integer RESTOCKABLE_2 = 10;
+
+    private final ZonedDateTime DATE_1 = ZonedDateTime.of(2017, 4, 1, 10, 0, 0, 0, ZoneId.of("UTC"));
+    private final ZonedDateTime DATE_2 = ZonedDateTime.of(2017, 5, 1, 20, 0, 0, 0, ZoneId.of("UTC"));
+
+    private List<InventoryEntryDraft> drafts;
+    private List<InventoryEntry> inventoriesDB;
+    private List<Channel> channelsDB;
+
+    {
```

Comment:
For tests initialization it is common convention to use `@BeforeClass`/`@Before`/`@AfterClass`/`@After` instead. These _instant initialization_  blocks are strongly discouraged.